src/__tests__/LDClient-events-test.js

```diff
@@ -1,3 +1,4 @@
+// @ts-nocheck
```

Comment:
Added this ts-nocheck as typescript checking in eslint isn't happy in this file. Nothing changed, but the typescript version was updated. Basically just affects the IDE.